test/helpers/position_assertions.cc

```diff
@@ -1831,19 +1842,33 @@ void ApplyCodeActionAssertion::checkAll(
     const CodeAction &codeAction) {
     const auto &config = wrapper.config();
     UnorderedMap<string, string> accumulatedOriginalEditedContents{};
+
+    // actualEditedFileContents -> (expectedUpdatedFilePath, expectedEditedFileContents)
+    // Maps original file contents to a edited filename and contents
+    UnorderedMap<string, std::pair<std::string, std::string>> fileToUpdatedFile;
     string actualEditedFileContents;
-    auto maybeFile = expectedFile();
-    if (!maybeFile.has_value()) {
-        return;
-    }
-    auto [expectedUpdatedFilePath, expectedEditedFileContents] = maybeFile.value();
     for (auto &c : *codeAction.edit.value()->documentChanges) {
         auto file = getFileByUri(config, sourceFileContents, c->textDocument->uri);
 
         c = sortEdits(move(c));
         actualEditedFileContents = string(file->source());
 
         for (auto &e : c->edits) {
+            auto &td = c->textDocument;
+            auto filename = td->uri;
+            auto version = td->version;
+
+            if (!std::holds_alternative<double>(version)) {
+                continue;
+            }
```

Comment:
I do not know what this piece of code is attempting to do. As far as I can tell, in Sorbet the `version` field is always `null` and never a `double`, which means that the edits to this file in this PR effectively disable every one of our `rbedited` file snapshot tests.